tests/test_elastic.py

```diff
@@ -439,6 +439,65 @@ def test_search_partial(self):
         assert_equal(len(contribs['users']), 0)
 
 
+@requires_search
+class TestProjectSearchResults(SearchTestCase):
+    def setUp(self):
+        super(TestProjectSearchResults, self).setUp()
+        self.user = UserFactory(usename='Doug Bogie')
+
+        self.singular = 'Spanish Inquisition'
+        self.plural = 'Spanish Inquisitions'
+        self.possessive = 'Spanish\'s Inquisition'
+
+        self.project_singular = ProjectFactory(
+            title=self.singular,
+            creator=self.user,
+            is_public=True,
+        )
+
+        self.project_plural = ProjectFactory(
+            title=self.plural,
+            creator=self.user,
+            is_public=True,
+        )
+
+        self.project_possessive = ProjectFactory(
+            title=self.possessive,
+            creator=self.user,
+            is_public=True,
+        )
+
+        self.project_unrelated = ProjectFactory(
+            title='Cardinal Richelieu',
+            creator=self.user,
+            is_public=True,
+        )
+
+    def test_singular_query(self):
+        """Verify searching for singular term includes singular,
```

Comment:
As per COSdev, prefer descriptive method names and code comments to docstrings *in tests*.

https://cosdev.readthedocs.org/en/latest/process/testing.html#general-testing-guidelines

I'll make this change when I merge.